src/common/3gpp_23.003.h

```diff
@@ -28,4 +28,6 @@
 #ifndef FILE_3GPP_23_003_SEEN
 #define FILE_3GPP_23_003_SEEN
 
+const uint8_t SST_MAX_STANDARDIZED_VALUE = 127;
```

Comment:
You may choose preprocessor #define SST_MAX_STANDARDIZED_VALUE ...
or if you use const, you may follow C++ coding guideline k...